README.md

```diff
@@ -9,6 +9,21 @@ Refer to [MOSIP docs](https://docs.mosip.io/1.2.0/modules/datashare).
 ## APIs
 API documentation is available [here](https://docs.mosip.io/1.2.0/api).
 
+## Durian in standalone mode
+Durian application has dependency on the external services such as partner management service for policy verification and key management service for crypto operations such as signature computation, Encryption etc.

```

Comment:
pms should not be called as external service, may be we can call them as other MOSIP module services

Reply:
Done